README.md

```diff
@@ -127,3 +131,56 @@ To add a new module, use **scripts/ipfs/** as an example to:
 * Your computer can be a node too! It will mesh with the Pi's over your router. See the [cjdns repository](https://github.com/cjdelisle/cjdns) on how to set this up.
 
 * Original plan for this repository and early benchmark results are available in [the doc folder](https://github.com/tomeshnet/prototype-cjdns-pi/blob/master/docs/).
+
+## FAQ
+
+**Q:** Why do my Orange Pi Zero USB headers not work?
+
+**A:** Some images are missing the USB overlay.  Simply add the following to the /boot/armbianEnv.txt file and restart the pi.
```

Comment:
We bold dirs in this repo. See: https://github.com/tomeshnet/prototype-cjdns-pi/pull/123/files#diff-04c6e90faac2675aa89e2176d2eec7d8R126